app/controllers/api/v1/classifications_controller.rb

```diff
@@ -36,10 +36,23 @@ def build_resource_for_update(update_params)
     classification
   end
 
+  def sms_ids_from_links(link_params)
+    sms = if ids = link_params.delete(:set_member_subjects)
+      SetMemberSubject.select(:id).find(ids)
```

Comment:
Meh - if we get the raise for free it really depends on the object creation time then after DB retrieval. I'll it up to you.